incidents/2021-11-15-alb-runs-out-of-nodes.md

```diff
@@ -0,0 +1,96 @@
+# 2021-11-15: ALB runs out of functioning nodes
+
+## description
+
+The ALB removed nodes from it's Target Group until only one remained and then that remaining node ran out of credits 
+after ~3 days. 
+
+## timeline
+
+UTC/ACDT
+
+03:16/13:46 - New Relic reports "Error Percentage (High)"
+
+* I click the acknowledge button and go investigating any spikes in traffic.
+    - these errors are typically enthusiastic bots, or a confluence of bots
+* I'm not logged in to NewRelic so it was never acknowledged.
+* traffic seems normal. I block a couple of random bots that are iterating through the corpus just in case with no effect.
+
+03:19/13:49 - New Relic reports "Journal (bypass CDN)"
+
+* I acknowledge again and log in to NewRelic this time.
+    - these are typically more serious
```

Comment:
I see the bypass CDN messages every now and again, but I've never worried about them much just assuming that it meant somebody has accessed a service directly, e.g. 'https://prod--journal.elifesciences.org/'.

Do we have a chart or table that details the types of alerts that we have and shows typically how serious they are? If not, that could be a useful guide to have (I'd find it useful anyhow). I'll raise a ticket and we can discuss when we next meet.

Reply:
> but I've never worried about them much

yeah, sometimes it's just a sign that the journal nodes are under pressure which may ease up in a few seconds or minutes.

Reply:
> Do we have a chart or table that details the types of alerts that we have and shows typically how serious they are? 

not in the sense that I think you're asking for. 

The alerts can be found in NewRelic under "Alerts & AI" -> "Alert Conditions (Policies)" as well as under "Infrastructure" -> "Settings".

It's up to the human to decide how severe the alert is.